tests/integration_tests_rust.rs

```diff
@@ -271,26 +271,45 @@ fn start_stop_reinit() {
 #[test]
 fn onchain_spend_receive() {
 	let (bitcoind, electrsd) = setup_bitcoind_and_electrsd();
-	let chain_source = TestChainSource::Esplora(&electrsd);
+	let chain_source = TestChainSource::BitcoindRpc(&bitcoind);
```

Comment:
Just checking to see if you want to change this. IIRC, there was some flakiness issues with `BitcoindRpc`?